test/3_TestWETH9.js

```diff
@@ -81,7 +81,7 @@ contract('WETH9', (accounts) => {
             // Perfom a deposit of the malicious amount
             await myWeth9Instance.deposit({'value': malicious_amount});
         } catch (err) {
-            console.log('       # ', err.message);
+            console.log('       (expect)  ', err.message);
```

Comment:
Look at this example on how to test an event https://github.com/gnosis/safe-contracts/blob/0941fef007c2d9cb6a37e4b9fca5b7e65a486a44/test/gnosisSafeDeploymentViaCreate2.js#L56